### PR TITLE
two_sum implementation and writes, passes tests

### PR DESCRIPTION
**Pull request by sarah-kw**
two_sum function code review
--take in list and target
-- return pairs of numbers in list that sum to target

### CODE REVIEW

File: functions/two_sum.py [hunk 1]
```diff
@@ -1,2 +1,5 @@
 def twoSum(n, t):
-    pass
+    for i in range(len(n)):
```
Comment (nikkitorab) [inline functions/two_sum.py:2]: add code to handle invalid input

File: tests/two_sum_test.py [hunk 2]
```diff
@@ -1,2 +1,38 @@
 import pytest
 from functions.two_sum import twoSum
+
```
Comment (nikkitorab) [inline tests/two_sum_test.py:3]: add tests for edge cases

File: functions/two_sum.py [hunk 3]
```diff
@@ -1,2 +1,5 @@
...
+        for j in range(i + 1, len(n)):
+            if n[j] == t - n[i]:
+                return [i, j]
```
Comment (nikkitorab) [inline functions/two_sum.py:5]: return something if there are no two items in n that add up to t

Comment (sarah-kw) [reply functions/two_sum.py:5]: see commit [cf6ef08](https://github.com/nikkitorab/more-code-reviews/pull/1/commits/cf6ef08fb0faa4ffad5401cb8a27866e9e699f0c)
